client/my-sites/plans/jetpack-plans/product-store/featured-item-card/index.tsx

```diff
@@ -39,18 +39,21 @@ export const FeaturedItemCard: React.FC< FeaturedItemCardProps > = ( {
 						<p>
 							<span>{ featuredDescription }</span>
 							<br />
-							<Button
-								className="featured-item-card--learn-more"
-								onClick={ onClickMore }
-								href="#"
-								plain
-							>
-								{ translate( 'More about %(product)s', {
-									args: {
-										product: title,
-									},
-								} ) }
-							</Button>
+
+							{ onClickMore && (
```

Comment:
The "More about <product-name>" link that opens the lightbox should **not** be shown if `isOwned`, `isIncludedInPlanOrSuperceded`,  and `isDeprecated`.  But don't quote me on those props exactly, but basically we do not want the user to be able to open the lightbox (which takes the user to checkout) in certain cases when we dont want the user to purchase the product (for example, if they already own the product in some way, or it is disabled somehow, or it is an external product (Like CRM is external product. Ideally the CRM product might still have a "More about product" link, but it will point to the external product page, not the lightbox - but this is the only exception I think.)

Reply:
> The "More about " link that opens the lightbox should **not** be shown if `isOwned`, `isIncludedInPlanOrSuperceded`, and `isDeprecated`. But don't quote me on those props exactly, but basically we do not want the user to be able to open the lightbox (which takes the user to checkout) in certain cases when we dont want the user to purchase the product (for example, if they already own the product in some way, or it is disabled somehow, or it is an external product (Like CRM is external product. Ideally the CRM product might still have a "More about product" link, but it will point to the external product page, not the lightbox - but this is the only exception I think.)

thanks @elliottprogrammer , I have updated the component now to make the link hidden if the product is 'isOwned', 'isIncludedInPlanOrSuperceded' or 'isDeprecated'.  I also created a dedicated MoreInfoLink component so it can handle the external products link such as the CRM.